api/routes/generate.js

```diff
@@ -47,149 +42,187 @@ exports.buildCrxConfig = function *(targetUrl) {
         filename: parsedUrl.hostname + '.crx'
     };
 
-    // Prepare request (send fake browser user-agent header)
-    var req = {
-        url: crxConfig.url,
-        headers: {
-            'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36'
-        }
-    };
-    
-    // Page HTML response
-    var res;
-    
+
     // Execute GET request to provided URL
     // May fail for internal URLs, continue anyway
     try {
-        res = yield thunkify(request)(req);
+        var dom = yield exports.getPageDom(crxConfig.url);
```

Comment:
You initialized `dom` inside a try-catch closure statement, which makes it unavailable outside of it. Did you run the code to make sure it works after your changes?